app/main/helpers/briefs.py

```diff
@@ -0,0 +1,30 @@
+from flask import abort
+
+
+def is_supplier_eligible_for_this_brief(data_api_client, supplier_id, brief_id):
+
+    # TODO: check supplier is eligible
+    return True
+
+
+def has_supplier_already_submitted_a_brief_response(data_api_client, supplier_id, brief_id):
+
+    brief_responses = data_api_client.find_brief_responses(brief_id=brief_id, supplier_id=supplier_id)['briefResponses']
+    return len(brief_responses) == 0
+
+
+def get_live_brief(data_api_client, brief_id):
+
+    brief = data_api_client.get_brief(brief_id)['briefs']
+    if brief['status'] != 'live':
+        abort(404)
+
+    return brief
+
+
+def inject_yes_no_questions_into_section_questions(section, brief):
+
+    # pass all of the brief yes/no questions into the appropriate ContentQuestions
+    for question in section.questions:
+        if question.type == 'boolean_list':
+            question.boolean_list_questions = brief[question.id]
```

Comment:
Brought it up with @allait and it doesn't really seem like we would get a lot from this at this juncture.  
This feels really particular to this particular field in this particular app, and it's also bound pretty tightly with the `boolean_list` macro (which looks for the `boolean_list_questions` attribute).

Passing a brief into a section also seems a bit funny.

Basically, we only need this for one field and we're only using this field in one place, and it feels like moving this code doesn't give us a useful generalisation.